.env

```diff
@@ -0,0 +1,7 @@
+MDBOOK_VERSION=0.4.6
+MDBOOK_LINKCHECK_VERSION=0.7.2
+MDBOOK_TOC_VERSION=0.6.1
+
+GIT_DEPLOY_DIR=book   
+GIT_DEPLOY_BRANCH=gh-pages
+GIT_DEPLOY_REPO=https://github.com/simonsan/std-dev-guide
```

Comment:
Will change that, before merging, just for now for testing purpose for the output on https://simonsan.github.io/std-dev-guide/